sklearn/tests/test_isotonic.py

```diff
@@ -680,3 +681,24 @@ def test_get_feature_names_out(shape):
     assert isinstance(names, np.ndarray)
     assert names.dtype == object
     assert_array_equal(["isotonicregression0"], names)
+
+
+def test_isotonic_regression_output_predict():
```

Comment:
Don‘t we have a common test for this?

Reply:
Apparently not but `IsotonicRegression` has a special tag `1darray` and I would not be surprised that we skip most of the test.